spark/src/main/scala/geotrellis/spark/io/accumulo/AccumuloRasterCatalog.scala

```diff
@@ -1,19 +1,16 @@
 package geotrellis.spark.io.accumulo
 
+import com.typesafe.config.ConfigFactory
```

Comment:
Why are these imports reordered?

If your IDE is doing it, please tell it to stop :) We have a method to the madness of imports, geotrellis ones up top, then other libraries, usually java or scala at the bottom.